.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+    tags: ["*.*.*"]
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Build
+        run: |
+          make dist GORELEASER_ARGS="${{ !startsWith(github.ref, 'refs/tags/') && '--snapshot' || '' }}"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Packagecloud upload
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          gem install --no-document --user-install --bindir ~/.local/bin package_cloud
+          for f in dist/*.deb; do
+              ~/.local/bin/package_cloud push ${{ github.repository }}/any/any "$f"
```

Comment:
This (and the same rpm line below) assumes the same username/projectname structure at packagecloud as the GH repo has. So `scop/ssh-inscribe` for my fork, `aakso/ssh-inscribe` for the real thing.

What would be needed after/if this is merged is setting up a packagecloud account by name aakso, setting up a ssh-inscribe repo there, grabbing the packagecloud API token (available from API settings) and storing in the project's Actions repository secrets as `PACKAGECLOUD_TOKEN`.

The naming could be changed if there are some other preferences.